src/api/routes/category.js

```diff
@@ -0,0 +1,16 @@
+import { Router } from 'express';
+import {
+  getCategories,
+  getRecipesByCategory,
+} from '../controllers/category/index.js';
+// import { auth, imageUpload } from '../middlewares/index.js';
+
+const router = Router();
+
+// CATEGORY
+router.get('/cotegory', getCategories);
```

Comment:
misspell ❌

Reply:
Don't you know cotgeory? It's a different type of category. :D

Reply:
If you know you know 😅